okhttp/src/main/java/okhttp3/Request.kt

```diff
@@ -109,7 +109,29 @@ class Request internal constructor(
       level = DeprecationLevel.WARNING)
   fun cacheControl(): CacheControl = cacheControl
 
-  override fun toString(): String = "Request{method=$method, url=$url, tags=$tags}"
+  override fun toString() = buildString {
+    append("Request{method=")
+    append(method)
+    append(", url=")
+    append(url)
+    if (headers.size != 0) {
```

Comment:
Annoyed this doesn't have `isEmpty` / `isNotEmpty`. Probably coming soon from me. This should behave more like a collection!